### DISCUSSION TITLE
帮帮我

### DISCUSSION THREAD

**Discussion by lhfwhahuwq** [github123666/cidaren#21] [Q&A]
配置config.json是什么意思？
怎么做

**Comment by lihuhuhu** [comment]
> 配置config.json是什么意思？ 怎么做

双击打开config.json文件

**Comment by lihuhuhu** [comment]
> 配置config.json是什么意思？ 怎么做

将code或token填入对应**引号**内部

**Comment by lhfwhahuwq** [comment]
code是把011RX那条放进去吧？

**Comment by lhfwhahuwq** [comment]
然后呢

**Comment by lihuhuhu** [comment]
ctrl + s 运行exe, 注意code是取等于后面&前面的值

**Comment by lhfwhahuwq** [comment]
exe是哪个来着

**Comment by lihuhuhu** [comment]


**Reply by lhfwhahuwq** [reply]
我找不到啊

**Reply by lhfwhahuwq** [reply]
他说code已经被使用请重新抓取，怎么办

**Reply by lhfwhahuwq** [reply]
token已过期怎么办。
是放在
![屏幕截图 2024-05-21 153116](https://github.com/github123666/cidaren/assets/170232406/960c5609-7afe-4942-ada1-86f33f15566a)
这个里面吧？然后就运行github123666那个？抓取的code在configjson中填在token里面就是token已过期，填code那里就是重新抓取

**Reply by lhfwhahuwq** [reply]
你看这
![屏幕截图 2024-05-21 162018](https://github.com/github123666/cidaren/assets/170232406/6875b2be-d2ca-47ba-8298-28ec6c8668a7)

**Reply by lhfwhahuwq** [reply]
这个图我懂的。
我没点击，悬浮看链接然后识别的，识别完对了一下，内容都一样（眼神没问题）。
我从开始到进config那里都是正确的吧
先进词达人看code    然后复制进config.json里。然后是不是就能运行GitHub123666了？
好兄弟，再帮我一次！

**Reply by lhfwhahuwq** [reply]
兄弟

**Reply by lhfwhahuwq** [reply]
这个在哪里
![屏幕截图 2024-05-22 203240](https://github.com/github123666/cidaren/assets/170232406/0da343ed-7b79-4ab3-8ae9-b2d21d3c73c3)

**Comment by lhfwhahuwq** [comment]
是不是github123666那个？

**Comment by lihuhuhu** [comment]


**Reply by lhfwhahuwq** [reply]
code是取等于后面&前面的值
那里面没code啊

**Reply by lhfwhahuwq** [reply]
NameError: name 'exit' is not defined
还有这个，怎么办

**Reply by lihuhuhu** [reply]
这边建议放弃,文档都不看

**Reply by lhfwhahuwq** [reply]
文档在哪

**Reply by lhfwhahuwq** [reply]
兄弟告诉我文档在哪
我去看

**Reply by lhfwhahuwq** [reply]
他那个自述文件是文档么？

**Reply by lhfwhahuwq** [reply]
兄弟

**Reply by lhfwhahuwq** [reply]
我用那个code要不要安装模型了？

**Reply by lhfwhahuwq** [reply]
那运行exe的时候我没找到你说的      code是取等于后面&前面的值啊

**Reply by lhfwhahuwq** [reply]
是这样，复制进去然后后面有没有操作了？

**Reply by lhfwhahuwq** [reply]
兄弟加我vx吧，我最后确认一下
好不好
a307806690

**Reply by lhfwhahuwq** [reply]
兄弟兄弟

**Reply by lhfwhahuwq** [reply]
token已过期怎么办

**Reply by lhfwhahuwq** [reply]
要不要下载python

**Comment by lhfwhahuwq** [comment]
原谅我兄弟
